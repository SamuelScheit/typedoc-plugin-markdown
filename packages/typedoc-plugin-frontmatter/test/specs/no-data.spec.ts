import * as fs from 'fs';
import * as path from 'path';

describe(`No Frontmatter Data`, () => {
  test(`should not write any YAML to page if no data found`, async () => {
    const contents = fs
      .readFileSync(
        path.join(__dirname, '../out/no-data/interfaces/SomeInterface.md'),
      )
      .toString();
    expect(contents).toMatchSnapshot();
  });
});
