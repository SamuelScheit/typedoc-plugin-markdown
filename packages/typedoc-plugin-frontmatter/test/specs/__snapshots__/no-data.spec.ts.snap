// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`No Frontmatter Data should not write any YAML to page if no data found 1`] = `
"# Interface: SomeInterface

## Tag One

0

## Tag Two

'Frontmatter' tag
"
`;
